src/nvim/sign.c

```diff
@@ -598,18 +600,15 @@ static void sign_define_cmd(char *name, char *cmdline)
       numhl = arg + 6;
     } else {
       semsg(_(e_invarg2), arg);
-      failed = true;
-      break;
+      return;
```

Comment:
?

Reply:
? This was done to free the allocated arguments strings which were removed in #25724. So we can return instead of breaking.

Reply:
Oh right.